pkg/cmd/taskrun/logs.go

```diff
@@ -68,43 +68,47 @@ tkn taskrun logs -f foo -n bar
 				return err
 			}
 
-			opts.streamer = pods.NewStream
-
-			return opts.run()
+			return opts.Run()
 		},
 	}
 
-	c.Flags().BoolVarP(&opts.allSteps, "all", "a", false, "show all logs including init steps injected by tekton")
-	c.Flags().BoolVarP(&opts.follow, "follow", "f", false, "stream live logs")
+	c.Flags().BoolVarP(&opts.AllSteps, "all", "a", false, "show all logs including init steps injected by tekton")
+	c.Flags().BoolVarP(&opts.Follow, "follow", "f", false, "stream live logs")
 
 	_ = c.MarkZshCompPositionalArgumentCustom(1, "__tkn_get_taskrun")
 	return c
 }
 
-func (lo *LogOptions) run() error {
-	if lo.taskrunName == "" {
+func (lo *LogOptions) Run() error {
+	if lo.TaskrunName == "" {
 		return fmt.Errorf("missing mandatory argument taskrun")
```

Comment:
Can we change the error message here to `"missing mandatory argument taskrun name"`?

Reply:
Done